src/typeahead/getCmsSuggestions.ts

```diff
@@ -0,0 +1,27 @@
+import { TypeAheadSuggestion, TypeAheadSuggestionContent } from '.'
+import { getCmsFromElasticSearch, getValuesFromES } from '../es/cms'
+import { CmsTypes, LABELS } from '../graphql/search/cms/config'
+
+export async function getCmsSuggestions(
+  query: string,
+  limit: number,
+): Promise<TypeAheadSuggestion[]> {
+  const types = [CmsTypes.Article, CmsTypes.Publication]
+  const { results, totalCount } = await getCmsFromElasticSearch({ q: query, types })
+
+  const formattedResults = results.map(({ _source: result }) => {
+    const { title, field_short_title: shortTitle, type, uuid } = getValuesFromES(result) as any
```

Comment:
ANY?! 😱

Reply:
Yeah, a lot of code in this repo is not properly typed. This came from the existing code so I consider it out of scope.